packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -451,6 +451,14 @@ class WebpackCLI {
                 ],
                 description: 'Print compilation progress during build.',
             },
+            {
+                name: 'interactive',
+                usage: '--interactive',
+                type: Boolean,
+                alias: 'i',
+                multiple: false,
+                description: 'Enable webpack interactive mode',
+            },
```

Comment:
Ideally we should support two arguments:
- `--interactive`, just clean screen before compilation and output stats (no extra information on screen)
- `--interactive=verbose`, like above + interface with information about keys and what they do

Reply:
Done